.gitignore

```diff
@@ -30,3 +30,4 @@ codegen/
 
 # Octave session info
 octave-workspace
+
```

Comment:
Are you using an auto-code formatter? My linter always prefers a newline at the end of the files too, but @jennyknuth actually removed the extra newline at the end in her last commit :) Maybe a Python/javascript preference difference. I'm indifferent either way.

Reply:
I use an auto-formatter for python, but I've never worried about the eof newline issue for .gitignore files. I'm not picky about this particular topic.

Reply:
Ditto, but I might have just added this particular commit directly from the GitHub editor, so not sure what it did. If this persists as a problem, and we have competing editors, we can address it.